cfbs/index.py

```diff
@@ -235,6 +250,26 @@ def save(self, data=None):
     def _module_is_in_build(self, module):
         return module["name"] in (m["name"] for m in self["build"])
 
+    def validate_added_module(module):
+        """Try to help the user with warnings in appropriate cases"""
+
+        name = module["name"]
+        if name.startswith("./") and name.endswith(".cf"):
+            assert os.path.isfile(name)
+            if not _has_autorun_tag(name):
+                log.warning("No autorun tag found in policy file: '%s'" % name)
+                log.warning("Tag the bundle(s) you want evaluated:")
+                log.warning('  meta: "tags" slist => { "autorun" };')
+            return
+        if name.startswith("./") and name.endswith("/"):
+            assert os.path.isdir(name)
+            policy_files = list(find(name, extension=".cf"))
+            with_autorun = [x for x in policy_files if _has_autorun_tag(x)]
+            if len(policy_files) > 0 and len(with_autorun) == 0:
```

Comment:
I'd do `if any(policy_files) and not any(with_autorun)` and used a generator instead of list for `with_autorun`, but YMMV.

Reply:
Oki.